packages/react-composites/src/composites/CallComposite/ConfigurationScreen.tsx

```diff
@@ -37,7 +37,9 @@ export const ConfigurationScreen = (props: ConfigurationScreenProps): JSX.Elemen
       <Stack horizontal wrap horizontalAlign="center" verticalAlign="center" tokens={configurationStackTokens}>
         <LocalPreview />
         <Stack>
-          <div className={titleContainerStyle}>{title}</div>
+          <div data-ui-id="call-composite-configuration-page-title" className={titleContainerStyle}>
```

Comment:
This should be injected via `useIdentifiers()`

Reply:
The main reason for not inlining this is that it becomes part of the public API. If it's there, Contoso can use it to select the element, and then we have to keep it forever.

Reply:
Hmmm `useIdentifiers()` in only for components scope. There are instances where `data-ui-id` attribute is directly assigned to the composite like https://github.com/Azure/communication-ui-library/blob/a311b0f86dddfb868b7e5c527aa15f0bb9a65d35/packages/react-composites/src/composites/CallComposite/LocalPreview.tsx#L68. Maybe we should move `IdentifierProvider` to react-composites but some elements in our components (like MessageThread) might be hard to verify.

Reply:
can we have a useCompositeIdentifiers?

Reply:
Never mind. Changed the tests to snapshot tests so I reverted the addition of data-ui-id's.